tests/network/service_mesh/test_service_mesh.py

```diff
@@ -6,7 +6,10 @@
 )
 from tests.network.utils import assert_authentication_request
 
-pytestmark = pytest.mark.service_mesh
+pytestmark = [
+    pytest.mark.service_mesh,
+    pytest.mark.usefixtures("label_istio_injection_namespace"),
```

Comment:
Is there a difference between setting the fixture here or on definition to specify `autouse=True`?

Reply:
Accepted.